src/app/components/Transcript/index.styles.tsx

```diff
@@ -0,0 +1,87 @@
+import { css, Theme } from '@emotion/react';
+import pixelsToRem from '#app/utilities/pixelsToRem';
+import { focusIndicatorThickness } from '../ThemeProvider/focusIndicator';
+
+export default {
+  transcript: ({ spacings, palette, isDarkUi }: Theme) =>
+    css({
+      backgroundColor: isDarkUi ? palette.GREY_7 : palette.WHITE,
+      display: 'block',
+      marginBottom: `${spacings.TRIPLE}rem`,
+      border: `solid ${pixelsToRem(3)}rem transparent`,
+    }),
+
+  summary: ({ spacings, palette }: Theme) =>
+    css({
+      padding: `${spacings.DOUBLE}rem`,
+      '&:hover, &:focus': {
+        cursor: 'pointer',
+        span: {
+          textDecoration: 'underline',
+        },
+      },
+      '&:focus-visible': {
+        outline: `${focusIndicatorThickness} solid ${palette.BLACK}`,
+        outlineOffset: `-${pixelsToRem(6)}rem`,
```

Comment:
Not sure whether this is the best approach. Using a negative outlineOffset due to the touch target. Will also need to consider darkUi styles. Alternative is wrapping the content inside of the summary in a span to target on focus-visible.

Reply:
FYI We will need to set the colour of this per UI background or use the double outline where we have both black and white (currenlty the dark ui has a black focus indicator).